src/cljs/ui/splash/page.cljs

```diff
@@ -13,81 +13,156 @@
             [ui.router.component :refer [page]]
             [ui.router.subs :as router.subs]
             [ui.subscriptions :as subs]
             [ui.utils :as utils :refer [<sub >evt]]))
 
+(def email-pattern #"[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?")
+
 (def use-styles (styles/make-styles (fn [_]
-                                      {:root       {:background "#ECEFF8"
-                                                    :min-width  "100%"
-                                                    :min-height "100vh"}
-                                       :card       {:width         "720px"
-                                                    :height        "300px"
-                                                    :box-shadow    "0px 30px 60px #313B5833"
-                                                    :border-radius "20px"
-                                                    :opacity       1}
-                                       :typography {:width          "134px"
-                                                    :height         "35px"
-                                                    :text-align     "left"
-                                                    :font           "normal normal 900 30px/35px Roboto"
-                                                    :letter-spacing "3.9px"
-                                                    :color          "#3A3668"
-                                                    :text-transform "uppercase"}
-                                       :centered   {:display         "flex"
-                                                    :flex-direction  "row"
-                                                    :justify-content "center"}
-                                       :middle     {:position  "absolute"
-                                                    :top       "50%"
-                                                    :left      "50%"
-                                                    :transform "translate(-50%, -50%)"}
-                                       :button     {:width          "324px"
-                                                    :height         "48px"
-                                                    :background     "#F0F0F0 0% 0% no-repeat padding-box"
-                                                    :border         "1px solid #F0F0F0"
-                                                    :border-radius  "8px"
-                                                    :textTransform  "none"
-                                                    :font           "normal normal medium 16px/27px Roboto"
-                                                    :letter-spacing "0px"
-                                                    :color          "#757575"}})))
+                                      {:grid {:background "#ECEFF8"
```

Comment:
Since we did a complete rewrite to move away from material-ui I don't think we should be adding more of it, maybe we can use this instances to little by little replace it or we can create a separate issue just to get rid of it so we don't have multiple ways of doing the same thing

Reply:
done